script.js

```diff
@@ -85,4 +85,22 @@ function popup() {
 }
 
 dispProject.forEach((buttons) => { buttons.addEventListener('click', popup); });
-/* `````````` */
\ No newline at end of file
+/* `````````` */
+
+/*    ````````````  Validation  ````````````    */
+
+function validateEmail() {
+  const emailAddress = document.getElementById('email').value.trim();
+  const regularExp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  /* Source for the regural expression - https://www.javascripttutorial.net/javascript-dom/javascript-form/ */
+  document.forms[0].onsubmit = (e) => {
+    if (regularExp.test(emailAddress)) {
+      document.getElementById('error-message').innerHTML = '';
+    } else {
+      document.getElementById('error-message').innerHTML = 'Please check the syntax of your email again';
+      document.getElementById('error-message').style = 'color:blue; outline: 2px solid red';
```

Comment:
- [ ] This check is not working as expected, kindly try to use something like `email === email.toLowerCase()`. I am sure it would work fine. What I can see now is that it submits even when I have uppercase in the Email.